.code-samples.meilisearch.yaml

```diff
@@ -0,0 +1,866 @@
+        switch result {
+        case .success(let key):
+            print(key)
+        case .failure(let error):
+            print(error)
+        }
+    }
+get_settings_1: |-
+  client.getSetting(UID: "movies") { (result: Result<Setting, Swift.Error>) in
+      switch result {
+      case .success(let setting):
+          print(setting)
+      case .failure(let error):
+          print(error)
+      }
+  }
+update_settings_1: |-
+  let settings = Settings(...)
+  client.updateSetting(UID: "movies", settings) { (result: Result<Update, Swift.Error>) in
+      switch result {
+      case .success(let update):
+          print(update)
+      case .failure(let error):
+          print(error)
+      }
+  }
+reset_settings_1: |-
+  client.resetSetting(UID: "movies") { (result: Result<Update, Swift.Error>) in
+      switch result {
+      case .success(let update):
+          print(update)
+      case .failure(let error):
+          print(error)
+      }
+  }
+get_synonyms_1: |-
+  client.getSynonyms(UID: "movies") { (result: Result<[String: [String]], Swift.Error>) in
+      switch result {
+      case .success(let synonyms):
+          print(update)
+      case .failure(let error):
+          print(error)
+      }
+  }
+update_synonyms_1: |-
+  let synonym = Synonym()
+  client.updateSynonyms(UID: "movies", synonym) { (result: Result<Update, Swift.Error>) in
+      switch result {
+      case .success(let update):
+          print(update)
+      case .failure(let error):
+          print(error)
+      }
+  }
+reset_synonyms_1: |-
+  client.resetSynonyms(UID: "movies") { (result: Result<Update, Swift.Error>) in
+      switch result {
+      case .success(let update):
+          print(update)
+      case .failure(let error):
+          print(error)
+      }
+  }
+get_stop_words_1: |-
+  client.getStopWords(UID: "movies") { (result: Result<[String], Swift.Error>) in
+      switch result {
+      case .success(let stopWords):
+          print(stopWords)
+      case .failure(let error):
+          print(error)
+      }
+  }
+update_stop_words_1: |-
+  let stopWords: [String] = ["of", "the", "to"]
+  client.updateStopWords(UID: "movies", stopWords) { (result: Result<Update, Swift.Error>) in
+      switch result {
+      case .success(let update):
+          print(update)
+      case .failure(let error):
+          print(error)
+      }
+  }
+reset_stop_words_1: |-
+  client.resetStopWords(UID: "movies") { (result: Result<Update, Swift.Error>) in
+      switch result {
+      case .success(let update):
+          print(update)
+      case .failure(let error):
+          print(error)
+      }
+  }
+get_ranking_rules_1: |-
+  client.getRankingRules(UID: "movies") { (result: Result<[String], Swift.Error>) in
+      switch result {
+      case .success(let rankingRules):
+          print(rankingRules)
+      case .failure(let error):
+          print(error)
+      }
+  }
+update_ranking_rules_1: |-
+  let rankingRules: [String] = ["of", "the", "to"]
```

Comment:
There are not ranking rules 😉

Reply:
I think I copied them from the javascript library, now I am confused. What I should do?

Reply:
Oh! This probably was a mistake when you copied. Now it's fix, see [these lines](https://github.com/meilisearch/meilisearch-js/blob/de9f211fbf1987e2218f25740894d52fc2becaa4/.code-samples.meilisearch.yaml#L110-L120).